docs/draft/6-payloads.md

```diff
@@ -175,7 +192,14 @@ message StickerMessage {
 ##### Image content type
 
 A `ChatMessage` with `IMAGE` `Content/Type` MUST also specify the `payload` of the image
-and the `type`
+and the `type`.
+
+Clients MUST sanitize the payload before accessing its content, in particular: 
+- Clients MUST choose a secure decoder
+- Clients SHOULD strip metadata if present without parsing/decoding it
+- Clients SHOULD NOT add metadata/exif when sending an image file for privacy and security reasons
```

Comment:
Should this be `MUST NOT` instead?

Reply:
I'd sure prefer a must not, I wanted to leave some space for features that rely on optional EXIF